dashboard/app/controllers/gallery_activities_controller.rb

```diff
@@ -63,7 +63,7 @@ def set_gallery_activity
   # Never trust parameters from the scary internet, only allow the white list through.
   def gallery_activity_params
     params[:gallery_activity][:user_id] ||= current_user.id if params[:gallery_activity] && current_user
-    params.require(:gallery_activity).permit(:activity_id, :user_id)
+    params.require(:gallery_activity).permit(:activity_id, :level_source_id, :user_id)
```

Comment:
Because when we were populating `activity_id`, the activity has knowledge of the level. The level source does not have knowledge of the `level_id`, so I would think we want to populate that column (note the column exists).